packages/canvas/canvas-renderer/src/CanvasRenderer.ts

```diff
@@ -13,6 +13,7 @@ import type {
     RenderTexture,
     BaseRenderTexture,
 } from '@pixi/core';
+import type { CanvasExtract } from '@pixi/canvas-extract';
```

Comment:
This is not correct. CanvasRenderer.plugins.extract. Also, this creates a circular dependency. CanvasRenderer should know nothing about canvas-extract.

Reply:
It is inessential because `import type` used inside TS only, and striped in build process

Reply:
Yes, but the type imports will be used when we start generating types.